tests/litmus/func-ops/scalar.src.mlir

```diff
@@ -0,0 +1,30 @@
+// VERIFY
+
+func.func private @simpl(%v: f32) -> f32
+func.func private @simpl_i(%v: i32) -> i32
+
+func.func @assoc(%v1: f32, %v2: f32) -> f32 {
```

Comment:
This isn't assoc; it is that add is commutative

Reply:
Fixed. How come I confuse between these two :o